src/app/client/src/app/modules/core/services/user/user.service.ts

```diff
@@ -147,8 +150,9 @@ export class UserService {
       url: `${this.config.urlConFig.URLS.USER.GET_PROFILE}${this.userid}`,
       param: this.config.urlConFig.params.userReadParam
     };
-    this.learnerService.get(option).subscribe(
+    this.learnerService.getWithHeaders(option).subscribe(
       (data: ServerResponse) => {
+        this.captureServerDate(data);
```

Comment:
no need for function

Reply:
this needs seperare function as it has some if checks and code comments better if it is seperate